src/index.js

```diff
@@ -533,7 +533,7 @@ export default class InputNumber extends React.Component {
     if (typeof val === 'number') {
       result =
         ((precisionFactor * val + precisionFactor * step * rat) /
-        precisionFactor).toFixed(precision);
+          precisionFactor).toFixed(precision);
```

Comment:
Don't change this.

Reply:
The editor is automatically formatted.
removed.